File: app/src/main/java/com/nowfloats/manufacturing/API/model/AddBrochures/ActionData.java
package com.nowfloats.manufacturing.API.model.AddBrochures;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ActionData {

    @SerializedName("uploadpdf")
    @Expose
    private Uploadpdf uploadpdf;
    @SerializedName("title")
    @Expose
    private String title;

    public Uploadpdf getUploadpdf() {
        return uploadpdf;
    }

    public void setUploadpdf(Uploadpdf uploadpdf) {
        this.uploadpdf = uploadpdf;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

}

File: dashboard/src/main/java/com/dashboard/model/live/quickAction/ActionData.kt
package com.dashboard.model.live.quickAction


import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class ActionData(
  @SerializedName("action_item")
  var actionItem: ArrayList<QuickActionItem>? = null,
  @SerializedName("type")
  var type: String? = null,
) : Serializable